When running .NET build on Linux/Mac build does not complete successfully

### Checklist

- [x] I confirm there are no unresolved issues reported on the [Chocolatey Status page](https://status.chocolatey.org).
- [x] I have verified this is the correct repository for opening this issue.
- [x] I have verified no other issues exist related to my problem.
- [x] I have verified this is not an issue for a specific package.
- [x] I have verified this issue is not security related.
- [x] I confirm I am using **official**, and **not** unofficial, or modified, Chocolatey products.

### What You Are Seeing?

When running a Chocolatey.Cake.Recipe .NET build, builds are working correctly on Windows, but not on Mac and Linux.

In the past, when building on anything other than Windows, it was necessary to augment the build settings with a framework path override.

However, when switching to use .NET to build Chocolatey CLI, it was found that Mac and Linux builds are failing when this override is applied.

It is assumed that this override is no longer required, and it should be removed.

### What is Expected?

Builds across all OSes should run successfully.

### How Did You Get This To Happen?

1. Run a .NET build on GitHub Actions, using Chocolatey.Cake.Recipe
2. See that Windows completes successfully, but Mac and Linux fail

### User Story

As a developer using Chocolatey.Cake.Recipe to orchestrate my builds,
I want to be able to rely on the recipe building correctly across all supported OSes (Windows, Mac, Linux)
so that I can verify that things work correctly across these platforms

### System Details

* Operating System: Windows 11
* Windows PowerShell version: 5.1.26100.7920
* Chocolatey CLI Version: 2.6.0
* Chocolatey Licensed Extension version: 7.0.0
* Chocolatey License type: Business
* Terminal/Emulator: Windows Terminal


### Installed Packages

```bash
N/A
```

### Output Log

```bash
========================================
DotNetBuild
========================================
Executing task: DotNetBuild
Building src/chocolatey.sln
Build will use FrameworkPathOverride=/usr/lib/mono/4.5/ since not building on Windows.
Resolved tool to path /usr/share/dotnet/dotnet
Executing: /usr/share/dotnet/dotnet build "src/chocolatey.sln" --configuration Release --no-restore /property:Version=2.7.0-alpha0033 /property:TreatWarningsAsErrors=False /property:AssemblyVersion=2.7.0.0 /property:FileVersion=2.7.0.0 /property:AssemblyInformationalVersion=2.7.0-alpha-20260217-c4cf0a69 /property:Copyright=Copyright%20©%202017%20-%202026%20Chocolatey%20Software%2C%20Inc.%20Copyright%20©%202011%20-%202017%2C%20RealDimensions%20Software%2C%20LLC%20-%20All%20Rights%20Reserved. /property:FrameworkPathOverride=/usr/lib/mono/4.5/
/usr/share/dotnet/sdk/10.0.102/Microsoft.CSharp.CurrentVersion.targets(130,9): warning MSB3884: Could not find rule set file "MinimumRecommendedRules.ruleset". [/home/runner/work/choco/choco/src/Chocolatey.PowerShell/Chocolatey.PowerShell.csproj]
CSC : warning CS1616: Option 'CryptoKeyFile' overrides attribute 'System.Reflection.AssemblyKeyFileAttribute' given in a source file or added module [/home/runner/work/choco/choco/src/chocolatey.resources/chocolatey.resources.csproj]
  chocolatey.resources -> /home/runner/work/choco/choco/src/chocolatey.resources/bin/Release/net48/chocolatey.resources.dll
CSC : warning CS1616: Option 'CryptoKeyFile' overrides attribute 'System.Reflection.AssemblyKeyFileAttribute' given in a source file or added module [/home/runner/work/choco/choco/src/Chocolatey.PowerShell/Chocolatey.PowerShell.csproj]
  Chocolatey.PowerShell -> /home/runner/work/choco/choco/src/Chocolatey.PowerShell/bin/Release/net48/Chocolatey.PowerShell.dll
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: Found conflicts between different versions of "System.IO.Compression" that could not be resolved. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: There was a conflict between "System.IO.Compression, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" and "System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     "System.IO.Compression, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" was chosen because it was primary and "System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" was not. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on "System.IO.Compression, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" [/usr/lib/mono/4.5/System.IO.Compression.dll]. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /usr/lib/mono/4.5/System.IO.Compression.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/usr/lib/mono/4.5/System.IO.Compression.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             System.IO.Compression [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on or have been unified to "System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" []. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.common/3.4.3/lib/net472/Chocolatey.NuGet.Common.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.common/3.4.3/lib/net472/Chocolatey.NuGet.Common.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.common/3.4.3/lib/net472/Chocolatey.NuGet.Common.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.configuration/3.4.3/lib/net472/Chocolatey.NuGet.Configuration.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.librarymodel/3.4.3/lib/net472/Chocolatey.NuGet.LibraryModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: Found conflicts between different versions of "System.Net.Http" that could not be resolved. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: There was a conflict between "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [/usr/lib/mono/4.5/System.Net.Http.dll]. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /usr/lib/mono/4.5/System.Net.Http.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/usr/lib/mono/4.5/System.Net.Http.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             System.Net.Http [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on or have been unified to "System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" []. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
CSC : warning CS1616: Option 'CryptoKeyFile' overrides attribute 'System.Reflection.AssemblyKeyFileAttribute' given in a source file or added module [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/services/INugetService.cs(25,38): warning CS0618: 'ISourceRunner' is obsolete: 'This interface is deprecated and will be removed in v3.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileSourceSetting.cs(87,28): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileSettings.cs(60,28): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileFeatureSetting.cs(59,20): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileConfigSetting.cs(55,20): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileApiKeySetting.cs(51,20): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/registration/ChocolateyRegistrationModule.cs(75,41): warning CS0618: 'ISourceRunner' is obsolete: 'This interface is deprecated and will be removed in v3.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateySearchCommand.cs(138,30): warning CS0618: 'ListCommandConfiguration.OrderByPopularity' is obsolete: 'This property is deprecated and will be removed in version 3.0. Use the 'OrderBy' property instead.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs(90,26): warning CS0618: 'ListCommandConfiguration.ExplicitSource' is obsolete: 'This property is deprecated and will be removed in v3. Check if the top-level ExplicitSources property is set instead.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs(158,45): warning CS0618: 'ChocolateyListCommand._unsupportedArguments' is obsolete: 'Remove unsupported argument in V3!' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs(159,61): warning CS0618: 'ChocolateyListCommand._unsupportedIncludeRegistryProgramsArguments' is obsolete: 'Remove unsupported argument in V3!' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]

Build FAILED.

/usr/share/dotnet/sdk/10.0.102/Microsoft.CSharp.CurrentVersion.targets(130,9): warning MSB3884: Could not find rule set file "MinimumRecommendedRules.ruleset". [/home/runner/work/choco/choco/src/Chocolatey.PowerShell/Chocolatey.PowerShell.csproj]
CSC : warning CS1616: Option 'CryptoKeyFile' overrides attribute 'System.Reflection.AssemblyKeyFileAttribute' given in a source file or added module [/home/runner/work/choco/choco/src/chocolatey.resources/chocolatey.resources.csproj]
CSC : warning CS1616: Option 'CryptoKeyFile' overrides attribute 'System.Reflection.AssemblyKeyFileAttribute' given in a source file or added module [/home/runner/work/choco/choco/src/Chocolatey.PowerShell/Chocolatey.PowerShell.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: Found conflicts between different versions of "System.IO.Compression" that could not be resolved. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: There was a conflict between "System.IO.Compression, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" and "System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     "System.IO.Compression, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" was chosen because it was primary and "System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" was not. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on "System.IO.Compression, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" [/usr/lib/mono/4.5/System.IO.Compression.dll]. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /usr/lib/mono/4.5/System.IO.Compression.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/usr/lib/mono/4.5/System.IO.Compression.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             System.IO.Compression [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on or have been unified to "System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" []. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.common/3.4.3/lib/net472/Chocolatey.NuGet.Common.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.common/3.4.3/lib/net472/Chocolatey.NuGet.Common.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.common/3.4.3/lib/net472/Chocolatey.NuGet.Common.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.configuration/3.4.3/lib/net472/Chocolatey.NuGet.Configuration.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.librarymodel/3.4.3/lib/net472/Chocolatey.NuGet.LibraryModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packaging/3.4.3/lib/net472/Chocolatey.NuGet.Packaging.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: Found conflicts between different versions of "System.Net.Http" that could not be resolved. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277: There was a conflict between "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on "System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [/usr/lib/mono/4.5/System.Net.Http.dll]. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /usr/lib/mono/4.5/System.Net.Http.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/usr/lib/mono/4.5/System.Net.Http.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             System.Net.Http [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:     References which depend on or have been unified to "System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" []. [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:         /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:           Project file item includes which caused reference "/home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll". [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.protocol/3.4.3/lib/net472/Chocolatey.NuGet.Protocol.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.commands/3.4.3/lib/net472/Chocolatey.NuGet.Commands.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.credentials/3.4.3/lib/net472/Chocolatey.NuGet.Credentials.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.dependencyresolver.core/3.4.3/lib/net472/Chocolatey.NuGet.DependencyResolver.Core.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.packagemanagement/3.4.3/lib/net472/Chocolatey.NuGet.PackageManagement.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.projectmodel/3.4.3/lib/net472/Chocolatey.NuGet.ProjectModel.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
/usr/share/dotnet/sdk/10.0.102/Microsoft.Common.CurrentVersion.targets(2437,5): warning MSB3277:             /home/runner/.nuget/packages/chocolatey.nuget.resolver/3.4.3/lib/net472/Chocolatey.NuGet.Resolver.dll [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
CSC : warning CS1616: Option 'CryptoKeyFile' overrides attribute 'System.Reflection.AssemblyKeyFileAttribute' given in a source file or added module [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/services/INugetService.cs(25,38): warning CS0618: 'ISourceRunner' is obsolete: 'This interface is deprecated and will be removed in v3.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/registration/ChocolateyRegistrationModule.cs(75,41): warning CS0618: 'ISourceRunner' is obsolete: 'This interface is deprecated and will be removed in v3.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateySearchCommand.cs(138,30): warning CS0618: 'ListCommandConfiguration.OrderByPopularity' is obsolete: 'This property is deprecated and will be removed in version 3.0. Use the 'OrderBy' property instead.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs(90,26): warning CS0618: 'ListCommandConfiguration.ExplicitSource' is obsolete: 'This property is deprecated and will be removed in v3. Check if the top-level ExplicitSources property is set instead.' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs(158,45): warning CS0618: 'ChocolateyListCommand._unsupportedArguments' is obsolete: 'Remove unsupported argument in V3!' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Warning: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs(159,61): warning CS0618: 'ChocolateyListCommand._unsupportedIncludeRegistryProgramsArguments' is obsolete: 'Remove unsupported argument in V3!' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileSourceSetting.cs(87,28): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileSettings.cs(60,28): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileFeatureSetting.cs(59,20): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileConfigSetting.cs(55,20): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
Error: /home/runner/work/choco/choco/src/chocolatey/infrastructure.app/configuration/ConfigFileApiKeySetting.cs(51,20): error CS0433: The type 'HashCode' exists in both 'Microsoft.Bcl.HashCode, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/home/runner/work/choco/choco/src/chocolatey/chocolatey.csproj]
    12 Warning(s)
    5 Error(s)
```

### Additional Context

N/A

### Acceptance Criteria

- [ ] .NET Build running on Mac completes successfully
- [ ] .NET Build running on Linux completes successfully
- [ ] .NET Build running on Windows continues to run successfully

### Related Issues

None

--- a/Chocolatey.Cake.Recipe/Content/build.cake
+++ b/Chocolatey.Cake.Recipe/Content/build.cake
@@ -108,15 +108,6 @@ BuildParameters.Tasks.DotNetRestoreTask = Task("DotNetRestore")
                             .WithProperty("AssemblyInformationalVersion", BuildParameters.Version.InformationalVersion)
                             .WithProperty("Copyright", BuildParameters.ProductCopyright);
 
-    if (BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows)
-    {
-        var frameworkPathOverride = new FilePath(typeof(object).Assembly.Location).GetDirectory().FullPath + "/";
-
-        // Use FrameworkPathOverride when not running on Windows.
-        Information("Restore will use FrameworkPathOverride={0} since not building on Windows.", frameworkPathOverride);
-        msBuildSettings.WithProperty("FrameworkPathOverride", frameworkPathOverride);
-    }
-
     DotNetCoreRestore(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreRestoreSettings
     {
         Sources = BuildParameters.NuGetSources,
@@ -198,15 +189,6 @@ BuildParameters.Tasks.DotNetBuildTask = Task("DotNetBuild")
                             .WithProperty("FileVersion",  BuildParameters.Version.FileVersion)
                             .WithProperty("AssemblyInformationalVersion", BuildParameters.Version.InformationalVersion)
                             .WithProperty("Copyright", BuildParameters.ProductCopyright);
-
-        if (BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows)
-        {
-            var frameworkPathOverride = new FilePath(typeof(object).Assembly.Location).GetDirectory().FullPath + "/";
-
-            // Use FrameworkPathOverride when not running on Windows.
-            Information("Build will use FrameworkPathOverride={0} since not building on Windows.", frameworkPathOverride);
-            msBuildSettings.WithProperty("FrameworkPathOverride", frameworkPathOverride);
-        }
 
         DotNetCoreBuild(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreBuildSettings
         {
@@ -359,15 +341,6 @@ public void CopyBuildOutput()
                             .WithProperty("AssemblyInformationalVersion", BuildParameters.Version.InformationalVersion)
                             .WithProperty("Copyright", BuildParameters.ProductCopyright);
 
-                if (BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows)
-                {
-                    var frameworkPathOverride = new FilePath(typeof(object).Assembly.Location).GetDirectory().FullPath + "/";
-
-                    // Use FrameworkPathOverride when not running on Windows.
-                    Information("Publish will use FrameworkPathOverride={0} since not building on Windows.", frameworkPathOverride);
-                    msBuildSettings.WithProperty("FrameworkPathOverride", frameworkPathOverride);
-                }
-
                 foreach (var targetFramework in parsedProject.NetCore.TargetFrameworks)
                 {
                     Information("Running dotnet publish for {0}...", project.Path.FullPath);
@@ -421,15 +394,6 @@ public void CopyBuildOutput()
                             .WithProperty("FileVersion",  BuildParameters.Version.FileVersion)
                             .WithProperty("AssemblyInformationalVersion", BuildParameters.Version.InformationalVersion)
                             .WithProperty("Copyright", BuildParameters.ProductCopyright);
-
-                if (BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows)
-                {
-                    var frameworkPathOverride = new FilePath(typeof(object).Assembly.Location).GetDirectory().FullPath + "/";
-
-                    // Use FrameworkPathOverride when not running on Windows.
-                    Information("Publish will use FrameworkPathOverride={0} since not building on Windows.", frameworkPathOverride);
-                    msBuildSettings.WithProperty("FrameworkPathOverride", frameworkPathOverride);
-                }
 
                 foreach (var targetFramework in parsedProject.NetCore.TargetFrameworks)
                 {

--- a/Chocolatey.Cake.Recipe/Content/testing.cake
+++ b/Chocolatey.Cake.Recipe/Content/testing.cake
@@ -187,15 +187,6 @@ BuildParameters.Tasks.DotNetTestTask = Task("DotNetTest")
                                 .WithProperty("AssemblyInformationalVersion", BuildParameters.Version.InformationalVersion)
                                 .WithProperty("Copyright", BuildParameters.ProductCopyright);
 
-        if (BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows)
-        {
-            var frameworkPathOverride = new FilePath(typeof(object).Assembly.Location).GetDirectory().FullPath + "/";
-
-            // Use FrameworkPathOverride when not running on Windows.
-            Information("Restore will use FrameworkPathOverride={0} since not building on Windows.", frameworkPathOverride);
-            msBuildSettings.WithProperty("FrameworkPathOverride", frameworkPathOverride);
-        }
-
         var projectsToTest = new FilePathCollection();
 
         if (BuildParameters.TestExecutionType == "unit")

